src/main/java/org/intellij/sdk/codesync/Constants.java

```diff
@@ -138,6 +146,12 @@ private Notification() {
         public static final String UPGRADE_ORG_PRICING_PLAN = "To continue, please sign your organization up for the Team plan.";
         public static final String TRY_PRO_FOR_FREE = "To continue, please sign up for a 30-day free trial of Pro.";
         public static final String TRY_ORG_PRO_FOR_FREE = "To continue, please sign your organization up for a 30-day free trial of the Team plan.";
+
+        // Team alert related notification messages
+        public static final String TEAM_ACTIVITY_ALERT_HEADER_MESSAGE = "CodeSync | Check your team's activity!";
+        public static final String TEAM_ACTIVITY_ALERT_MESSAGE = "Hey, check your team's activity!";
+        public static final String TEAM_ACTIVITY_ALERT_SECONDARY_MESSAGE =
```

Comment:
This is what we are using on VSCode: https://github.com/codesyncapp/codesync-vs/blob/master/src/constants.ts#L107

Let me know if I should update on the VSCode side.